readme.md

```diff
@@ -95,7 +95,7 @@ const tasks = new Listr([
 
 ### Promises
 
-A `task` can also be async by returning a `Promise`. If the promise resolves, the task completed successfully, it it rejects, the task failed.
+A `task` can also be async by returning a `Promise`. If the promise resolves, the task completed successfully, it it rejects, the task failed. If the task function updates the task's `output` property it will be displayed.
```

Comment:
This is not specific to using promises though, it can be used everywhere like in Observables, Streams or just synchronously. So instead, a more detailed explanation should be added to [this](https://github.com/SamVerschueren/listr#task-object) section.